docs/sdk/api-reference.md

```diff
@@ -1181,6 +1161,44 @@ from clarifai.client.dataset import Dataset
 Dataset().export(save_path='output.zip')
 ```
 
+## Deployment
```

Comment:
Mohit, Deployment doc looks good to me but can you shift it to the end so that we can keep CO related at one place - Compute cluster, Nodepool, Deployment classes

Reply:
Sure

Reply:
Deployment section shifted to end